cli/src/main/java/org/commonjava/maven/ext/cli/Cli.java

```diff
@@ -541,4 +558,35 @@ private Settings parseSettings( File settings ) throws ComponentLookupException,
 
         return effectiveSettings;
     }
+
+    private class Header
+    {
+        private String name;
+
+        private String value;
+
+        public Header( String header )
+        {
+            String[] tokens = header.split( ":", 2 );
+            name = tokens[0].trim();
+            if ( tokens.length > 1 )
+            {
+                value = tokens[1].trim();
+            }
+            if ( value == null )
+            {
+                value = "";
+            }
+        }
+
+        public String getName()
+        {
+            return name;
+        }
+
+        public String getValue()
+        {
+            return value;
+        }
+    }
```

Comment:
Not required as per conversation on IRC

Reply:
> Not required as per conversation on IRC

This is used to split the headers into their components. Unless I am missing something, if we use a single property, then we still need to split the many headers into single headers, and then split the header into name-value pairs since Unirest expects a `java.util.Map` for multiple headers.

Reply:
Ok yes I would agree .... but the CLI shouldn't be doing it as it will already store the property - which I think I gave a potential suggestion for - more likely it would be inside RESTState.